app/services/vacancy_filters.rb

```diff
@@ -31,12 +31,12 @@ def to_hash
   end
 
   def only_active_to_hash
-    to_hash.delete_if { |_, v| v.blank? }
+    to_hash.delete_if { |_, v| v.blank? || v.nil? }
```

Comment:
Can we just do `v.empty?` for this?

Reply:
There is one method that handles both, after testing I think it is in fact `blank?` rather than `empty?` as the test suite is very unhappy when only using `empty?` so I've switched to `blank?` only!

<img width="703" alt="Screenshot 2019-03-19 at 09 55 43" src="https://user-images.githubusercontent.com/912473/54596863-692c3780-4a2d-11e9-8a58-3c1b5f49dba3.png">

<img width="497" alt="Screenshot 2019-03-19 at 09 56 23" src="https://user-images.githubusercontent.com/912473/54596867-6b8e9180-4a2d-11e9-8c06-5b3616e43105.png">